example/http/client/client.go

```diff
@@ -21,16 +21,17 @@ import (
 	"net/http"
 
 	"github.com/open-telemetry/opentelemetry-go/api/tag"
-	"github.com/open-telemetry/opentelemetry-go/api/trace"
+	apitrace "github.com/open-telemetry/opentelemetry-go/api/trace"
 	"github.com/open-telemetry/opentelemetry-go/plugin/httptrace"
 
 	"google.golang.org/grpc/codes"
 
 	_ "github.com/open-telemetry/opentelemetry-go/exporter/loader"
+	"github.com/open-telemetry/opentelemetry-go/sdk/trace"
```

Comment:
`google.golang.org/grpc/codes` should go up a section and the `github.com/open-telemetry/opentelemetry-go` sections should be joined

Reply:
done.